Use `fs::dir_create()` in intro notebooks (#699)
AlexsLemonade/training-modules: intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

dir.exists("results")
```

If the above says `FALSE` that means we will need to create a `results` directory.
We've previously seen that we can make directories in R using the base R function `dir.create()`.

> But, as we've also seen, this function will throw an error if you try to create a directory that already exists, which can be frustrating if you are re-running code!

Reviewer: Making this a bit more parallel (and no comma after initial "But")
```suggestion
But we've also seen that this function will throw an error if you try to create a directory that already exists, which can be frustrating if you are re-running code!
```

> A different option is to use the [`fs`](https://fs.r-lib.org/) package, which provides functions for you to interact with your computer's file system, and _bonus_ it's guaranteed to work on any kind of operating system (Max, Linux, PC, etc.)!

Reviewer: Is there any reason that we would expect dir.create() not to work on any operating sytem? This seems like FUD.
```suggestion
A different option is to use the [`fs`](https://fs.r-lib.org/) package, which provides functions for you to interact with your computer's file system with a more consistent behavior than the base R functions.
```

One function from this package is `fs::dir_create()` (note that it has an _underscore_, not a period), and much like the base R `dir.create()`, it creates directories. 
It has some other helpful features too:
- It will simply do nothing if that directory already exists; no errors, and nothing will get overwritten

> - It allows you to quickly create _nested_ directories, i.e. in one call make directories inside of other directories

Reviewer: So does dir.create? 
```suggestion
- It allows creating _nested_ directories by default, i.e. in one call make directories inside of other directories
```

Author: > So does dir.create?

Right just not by default, which is what I meant with the super duper non-specific "quickly"

> dir.exists("results")

Reviewer: Can't comment but the comment above should not say "re-check" anymore.

Author: I think it should? The first check was on line 82 (also hidden by default in this `files changed` though!).

Author: @jashapiro just want to make sure you've seen this comment before I head to merge

Author: @jashapiro quick ping for a quick look!

Reviewer: Yeah, it's fine.

Author: thanks, didn't want to miss something silly again like in the other dir_create PR :)
